src/middleware.js

```diff
@@ -80,7 +80,9 @@ Settings = (isAngel = true, otherBot) => async (ctx, next) => {
     ctx.otherBot = otherBot
     ctx.chatTarget = isAngel ? "Angel" : "Mortal"
     ctx.chatAs = isAngel ? "Mortal" : "Angel"
-
+    // TODO: Change this according to the different theme days
+    // FORMAT: [dayNum, monthNum]
+    ctx.themeDays = [[27, 8], [28, 8], [29, 8], [30, 8], [31, 8], [1, 9]]
```

Comment:
Could we put this as a json in the env file or in a config.js file? We shouldn't include magic constants in logic files